GitHub_LearningTests/Generic/OnboardingPersistancy/Resources/OnboardingResourceTests.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  OnboardingResourceTests.swift
+//  GitHub_LearningTests
+//
+//  Created by Amin Ashhab on 18.12.2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import XCTest
+import CleanCore
+import CleanPlatform
+@testable import GitHub_Learning
+
+class OnboardingResourceTests: XCTestCase {
+    private var sut: OnboardingResourceImpl!
+    
+    private func setUpTests(
+        localStorage: LocalStorage = LocalStorageDummy()
+    ) {
+        sut = OnboardingResourceImpl(localStorage: localStorage)
+    }
+    
+    func test_givenOnboardingResource_whenloadIsOnboardingFinished_thenDataAreLoaded() throws {
+        // Given
+        let localStorage = LocalStorageDummy()
+        setUpTests(localStorage: localStorage)
+        
+        // When
+        let _ = try sut.loadIsOnboardingFinished()
+        
+        // Then
+        XCTAssert(localStorage.dataLoaded == true)
+    }
+    
+    func test_givenOnboardingResource_whenStoreIsOnboardingFinished_thenDataAreLoaded() {
+        // Given
+        let localStorage = LocalStorageDummy()
+        let isFinished = true
+        setUpTests(localStorage: localStorage)
+        
+        // When
+        sut.storeIsOnboardingFinished(isFinished)
+        
+        // Then
+        XCTAssert(localStorage.isOnboardingFinished == isFinished)
```

Comment:
Tento test nedava smyls, protoze kdyby jsi nastail `isFinished` na false, tak ti to neprojde, i kdyz se ten test tvari ze jo.

Reply:
Bud kontroluj ze se to na tom vola a nebo primo tu hodnotu co se ulozila.